docs/source/query/overview.rst

```diff
@@ -0,0 +1,216 @@
+==============
+Querying Snuba
+==============
+
+This guide drives you through the process of authoring and testing a
+Snuba query.
+
+Exploring the Snuba data model
+==============================
+
+In order to architect a Snuba query, the first step is being able to
+know which Dataset you should query, which Entities you should select
+and what the schema of each Entity is.
+
+For an introduction about Datasets and entities, see the :doc:`/architecture/datamodel`
```

Comment:
```suggestion
For an introduction about Datasets and Entities, see the :doc:`/architecture/datamodel`
```

I put up a few capitalization changes. They're more nits than anything, but I think we should strive to be consistent in how we capitalize things.